manifests/c/com/exelearning/editor/2.7/com.exelearning.editor.locale.es-ES.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: com.exelearning.editor
+PackageVersion: 2.7
+PackageLocale: es-ES
+Publisher: eXelearning
+PackageName: eXelearning

```

```suggestion
PackageName: eLearning XHTML & HTML5 editor
```
Must match Control Panel / Apps&Features